src/graphql/journey.test.ts

```diff
@@ -1659,7 +1659,7 @@ describe('query cluster with consistency level', () => {
   });
 });
 
-describe('query with group by', () => {
+describe.skip('query with group by SKIPPED BECAUSE OF XREFS RETURN OPTIMISATION BUG', () => {
```

Comment:
Is there a GH issue link we can add here?

Reply:
Not right now, I communicated the problem with Dirk. I will not merge the `dev/1.27` branch until it is fixed in core or we have an issue tracking it. Just added this skip for this PR into `dev/1.27`